make/gen.cmake

```diff
@@ -111,6 +113,8 @@ function(V2X)
   )
 
   add_file_target(FILE "${V2X_NAME}.model.xml" GENERATED)
+  get_file_target(SRC_TARGET_NAME "${V2X_NAME}.model.xml")
+  set_target_properties(${SRC_TARGET_NAME} PROPERTIES INCLUDE_FILES "${INCLUDE_FILES}")
```

Comment:
And this should be the model XML include list, etc

Reply:
should be fixed now